src/components/PeoplePage.tsx

```diff
@@ -1,34 +1,198 @@
+    }
+
+    if (sex) {
+      filtered = filtered.filter((person) => person.sex === sex);
+    }
+
+    return filtered;
+  };
+
+  const sortPeople = (filteredPeople: Person[]) => {
+    let sortedPeople = [...filteredPeople];
+    let sortDirection = 1;
+
+    if (sort && order) {
+      sortDirection = -1;
+    }
+
+    switch (sort) {
+      case 'name':
+      case 'sex':
+        sortedPeople = sortedPeople
+          .sort((personA, personB) => (personA[sort]
+            .localeCompare(personB[sort]) * sortDirection));
+        break;
+      case 'born':
+      case 'died':
+        sortedPeople = sortedPeople
+          .sort((
+            personA,
+            personB,
+          ) => (personA[sort] - personB[sort]) * sortDirection);
+        break;
+      default:
+        break;
+    }
+
+    return sortedPeople;
+  };
+
+  const filterAndSortPeople = useCallback(() => {
+    const filteredPeople = filtrForPeople();
+    const sortedPeople = sortPeople(filteredPeople);
+
+    return sortedPeople;
+  }, [sort, query, sex, centuries, order]);
+
+  const isAnyMatch = !isInPorcess && filterAndSortPeople().length === 0;
+  const isAnyPeople = people.length === 0 && !isInPorcess;
+
   return (
     <>
       <h1 className="title">People Page</h1>
 
       <div className="block">
         <div className="columns is-desktop is-flex-direction-row-reverse">
           <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
+            {!isInPorcess
+              && (
+                <PeopleFilters
+                  searchParams={searchParams}
+                  centuries={centuries}
+                  query={query}
+                  sex={sex}
+                  onQueryChange={onQueryChange}
+                />
+              )}
           </div>
 
           <div className="column">
             <div className="box table-container">
-              <Loader />
+              {error
+                && (
+                  <p data-cy="peopleLoadingError" className="has-text-danger">
+                    Something went wrong
+                  </p>
+                )}
 
-              <p data-cy="peopleLoadingError">Something went wrong</p>
+              {isAnyPeople && (
+                <p data-cy="noPeopleMessage">
+                  There are no people on the server
+                </p>
+              )}
 
-              <p data-cy="noPeopleMessage">
-                There are no people on the server
-              </p>
+              {isAnyMatch && (
+                <p>
+                  There are no people matching the current search criteria
+                </p>
+              )}
 
-              <p>There are no people matching the current search criteria</p>
-
-              <PeopleTable />
+              {isInPorcess
+                ? <Loader />
+                : (filterAndSortPeople().length > 0 && !isInPorcess) && (
+                  <PeaopleTable
```

Comment:
There is a typo in the component name. It should be 'PeopleTable' instead of 'PeaopleTable'.